microshift_rest_api/index.adoc

```diff
@@ -0,0 +1,119 @@
+// Automatically generated by 'openshift-apidocs-gen'. Do not edit.
+:_content-type: ASSEMBLY
+[id="api-index"]
+= API index
+
+[cols="1,1",options="header"]
+|===
+^| API ^| API group
+| xref:./extension_apis/apiservice-apiregistration-k8s-io-v1.adoc#apiservice-apiregistration-k8s-io-v1[APIService]
```

Comment:
All of these likely have to be in the form of link:https://; for example,
link:https://access.redhat.com/solutions/60959[How to embed a Kickstart file into an ISO image].